src/utils/eventLevels.js

```diff
@@ -95,3 +94,33 @@ export function sortEvents(evtA, evtB, { startAccessor, endAccessor, allDayAcces
     || !!get(evtB, allDayAccessor) - !!get(evtA, allDayAccessor) // then allDay single day events
     || +get(evtA, startAccessor) - +get(evtB, startAccessor)     // then sort by start time
 }
+
+// Polyfill for IE11 
+if (!Array.prototype.findIndex) {
```

Comment:
Instead of adding the polyfill let's make a note in the docs for users to add it instead as part of the setup process. It's not great practice to alter global objects in libraries

Reply:
I really understand this point, but on the other hand I think it is not a good idea to exclude a whole browser because of this. I know the IE is deprecated with the release of Edge, but at the moment there are plenty of users out there using this hell of a software, because they don't want, can't or don't know how to switch to FF, Edge or Chrome. I think a lot of devs will miss the hint to use polyfills, since they don't notice they are necessary, because they don't test the IE. In the End it is up to you how to handle this, I just wanted to point out the issue we ran into, and why. Maybe there is another compromise to fix the issues mentioned in my request.

Reply:
I'm suggesting we exclude a browser and we should definately support Ie. I'm saying we should take the same approach as React which is specify what polyfills are required to use the component. I realize this may mean that folks may miss it initially but its better than being a bad citizen in the JS community by changing the global state, which can break other stuff in unexpected ways.

If you'd prefer we can just switch instances of the native findIndex to the lodash one, which is already included 👍 ?

Reply:
That would be ok for me, I guess :-)